products/flux2.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Flux
+category: server-app
+tags: cncf
+iconSlug: flux2
+permalink: /flux
+alternate_urls:
+-   /flux2
+-  /fluxcd
+versionCommand: flux version
+releasePolicyLink: https://fluxcd.io/flux/releases/#supported-releases
+changelogTemplate: https://github.com/fluxcd/flux2/releases/tag/v__LATEST__
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+-   git: https://github.com/fluxcd/flux2.git
+
+# eol(X) = releaseDate(X+3)
+releases:
+-   releaseCycle: "2.0"
+    releaseDate: 2023-07-05
+    eol: false
+    latest: "2.0.1"
+    latestReleaseDate: 2023-07-11
+
+---
+
+> [Flux](https://fluxcd.io) is an open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+
+For the CLI and its controllers the project support the last three minor releases. Critical bug fixes, such as security fixes, may be back-ported to those three minor versions as patch releases, depending on severity and feasibility.
```

Comment:
```suggestion
The project support the last three minor releases of the CLI and its controllers, with critical bug and security fixes.
```

IMO it's always depending on the severity / feasibility, I would suggest to drop this mention.